.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+- package-ecosystem: cargo
+  directory: "/"
+  schedule:
+    interval: weekly
```

Comment:
FWIW I'm not sold on the benefit/noise ratio of Dependabot, especially when manual work is required to get any _important_ upgrades like to `rustls` and other fundamental crates this repository relies on. This isn't a blocker though.

Reply:
I think it's useful for keeping the lockfile up-to-date with semver compatible updates. Yes, semver incompatible updates will always require manual work.

Reply:
I'm not sure we agree on the usefulness because our lockfile doesn't get distributed or matter to downstream consumers of the library, but my thoughts here aren't a blocker.

Reply:
Fair. We can back it out if it ends up being annoying.